essentials-framework/Essentials Devices Common/Essentials Devices Common/DSP/DspBase.cs

```diff
@@ -9,24 +9,34 @@
 
 namespace PepperDash.Essentials.Devices.Common.DSP
 {
-	public abstract class DspBase : EssentialsDevice
+	public abstract class DspBase : EssentialsDevice, IHasDspPresets
```

Comment:
Implementation of `IHasDspPresets` should be on the implementing class, not the base class

Reply:
I'm going to remove the implementation on the built-in tesira classes.

We should probably consider removing them anyway - my plugin works well and uses appropriate interfaces for essentials.